cmd/sourced/compose/workdir/workdir.go

```diff
@@ -134,7 +134,7 @@ func UnsetActive() error {
 		return err
 	}
 
-	_, err = os.Stat(dir)
+	_, err = os.Lstat(dir)
```

Comment:
could you explain why do you need to use `Lstat` here?

Reply:
from the description:
>_`UnsetActive` must work with `Lstat` to check symlink status,
instead of accessing the pointed dir._

From the function docs:
```
// UnsetActive removes symlink for active workdir
```

So let's consider:
```
__active__ -> ~/.sourced/workdirs/this/dir/was/deleted
```

Then `os.Stat(dir)` is going to return `os.ErrNotExist` (because it checks the pointed file, not the symlink itself), so `os.Remove(dir)` is not going to be called, so `__active__` is not going to be deleted.

Reply:
makes sense 👍